app/src/main/java/org/simple/clinic/patient/PatientSearchCriteria.kt

```diff
@@ -0,0 +1,8 @@
+package org.simple.clinic.patient
+
+sealed class PatientSearchCriteria {
+
+  data class ByName(val patientName: String): PatientSearchCriteria()
```

Comment:
'Criteria' already covers what this sealed class is.
I think we can rename this to just `Name` and `PhoneNumber`.
So it means that search criteria is name or phone number.

Reply:
I'll make this change.